platform-includes/logs/options/android.mdx

```diff
@@ -0,0 +1,46 @@
+#### beforeSendLog
+
+To filter logs, or update them before they are sent to Sentry, you can use the `getLogs().beforeSend` option.
+
+```java {tabTitle: Java}
+import io.sentry.SentryLevel;
+import io.sentry.android.core.SentryAndroid;
+import android.app.Application;
+
+public class MyApplication extends Application {
+  public void onCreate() {
+    super.onCreate();
+    SentryAndroid.init(this, options -> {
+      options.setDsn("https://examplePublicKey@o0.ingest.sentry.io/0");
+      options.getLogs().setBeforeSend((logEvent) -> {
+        // Modify the event here:
+        logEvent.setBody("new message body");
+        return logEvent;
+      });
+    });
+  }
+}
+```
+
+```kotlin {tabTitle: Kotlin}
```

Comment:
since it's a big code snippet it'd be nice to highlight the relevant parts (the beforeSend)

```suggestion
```kotlin {tabTitle: Kotlin} {11-15}
```

Reply:
maybe this also make sense for the other code snippets